docs/docs/pods.md

```diff
@@ -0,0 +1,583 @@
+---
+title: Pods
+---
+
+# Pods
+
+Marathon supports the creation and management of pods. Pods enable you to share storage, networking, and other resources among a group of applications on a single agent and address them as one group rather than as separate applications, similar to a virtual machine. Pods allow quick, convenient coordination between applications that need to work together, for instance a database and web server that make up a content management system.
+
+Currently, Marathon pods can only be created and administered via the `/v2/pods/` endpoint of the REST API, not via the web interface.
+
+# Features
+
+- Co-located containers.
+- Pod-level resource isolation.
+- Sandbox and ephemeral volumes that are shared among pod containers.
+- Configurable sandbox mount point for each container.
+- Pod-level health checks.
```

Comment:
this list is great and may be what you want to say in the intro...  pods provides an ability for 2 or more applications to share resources, health to be managed as a unit.